README.md

```diff
@@ -1,16 +1,43 @@
 # provider-terraform-vsphere
 
-An experimental @crossplane provider for @vmware vSphere
+
+## Overview
+
+`provider-terraform-vsphere` is an experimental Crossplane infrastructure provider for
+the vSphere [vCenter Server](https://www.vmware.com/products/vcenter-server.html) and
+[ESXi](https://www.vmware.com/products/esxi-and-esx.html) APIs.
+Available resources and their fields can be found in the [CRD
+Docs](https://doc.crds.dev/github.com/crossplane-contrib/provider-terraform-vsphere).
+Documentation for the original Terraform resources that were used to generate these CRDs,
+and descriptions for the attributes which compose them, can be found in the documentation for the
+[Terraform vSphere provider](https://registry.terraform.io/providers/hashicorp/vsphere/latest/docs).
+Source code for the Terraform provider can be found at https://github.com/hashicorp/terraform-provider-vsphere
+If you encounter an issue please reach out on
+[slack.crossplane.io](https://slack.crossplane.io) and create an issue in
+the [crossplane-contrib/provider-terraform-vsphere](https://github.com/crossplane-contrib/provider-terraform-vsphere)
+repo.
+
+* Custom Resource Definitions (CRDs) that model vSphere the infrastructure and services, following
+  the schema observed in the Terraform vSphere Provider.
+* Generated code enabling Go representations of the CRDs to be encoded and decoded to/from the native Terraform
+  format [zclconf/go-cty](https://github.com/zclconf/go-cty) for serialization to the Terraform GRPC wire format
+* Generated code enabling Go representations of Resources to be compared, so that differences between the desired
+  and observed state of a Resource can be detected.
+* Taken together, these capabilities enable the generic controller implemented in 
+  [crossplane-contrib/terraform-runtime](https://github.com/crossplane-contrib/terraform-runtime) to fulfill the
+  [crossplane/crossplane-runtime](https://github.com/crossplane/crossplane-runtime) `ExternalClient` interface.
 
 This provider was generated by running https://github.com/crossplane-contrib/terraform-provider-gen
 pointed at the plugin binary built from https://github.com/hashicorp/terraform-provider-vsphere
 
 The same provider binary used to build the provider must also be present at runtime. Point the runtime at the
 full path to the plugin on the local filesystem using the `--pluginPath=` cli flag. When using an official image
-the appropriate plugin binary will be bundled.
+the appropriate plugin binary will be bundled. For local development, the corresponding 
```

Comment:
Did this get cut off?

Reply:
I must have seen a squirrel. I'll fix that up.